android/ui/src/main/java/com/trafi/ui/theme/Color.kt

```diff
@@ -8,16 +8,7 @@ import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.graphics.Color
 import com.trafi.ui.theme.internal.ColorPalette
-
-@Stable val Grey100 = ColorPalette.Gray100
-@Stable val Grey200 = ColorPalette.Gray200
-@Stable val Grey300 = ColorPalette.Gray300
-@Stable val Grey400 = ColorPalette.Gray400
-@Stable val Grey500 = ColorPalette.Gray500
-@Stable val Grey600 = ColorPalette.Gray600
-@Stable val Grey700 = ColorPalette.Gray700
-@Stable val Grey800 = ColorPalette.Gray800
-@Stable val Grey900 = ColorPalette.Gray900
+import com.trafi.ui.theme.internal.GrayScale
```

Comment:
If we're exposing `GrayScale` to our SDK users, it should be in the `com.trafi.ui.theme` package (not `internal`)